decimal64.go

```diff
@@ -454,9 +453,32 @@ func (d Decimal64) IsInt() bool {
 		return false
 	}
 }
-func (d Decimal64) isZero() bool {
-	fl, _, _, significand := d.parts()
-	return significand == 0 && fl == flNormal
+
+// IsSubnormal returns true iff d is a subnormal.
+func (d Decimal64) IsSubnormal() bool {
+	flavor, _, exp, significand := d.parts()
+	return significand < decimal64Base && exp >= minExp && flavor == flNormal && significand != 0
+}
+
+// updateMag updates the magnitude of the dec object
+func (dec *decParts) isZero() bool {
+	return dec.significand.lo == 0 && dec.fl == flNormal
```

Comment:
What about `dec.significand.hi`?

Reply:
It is now being checked alongside `dec.significand.lo`